extensions/blocks/donations/editor.scss

```diff
@@ -0,0 +1,72 @@
+/**
+ * Editor styles for Donations
+ */
+.wp-block-jetpack-donations .donations__container {
+  border: 1px solid #a7aaad;
+
+  .donations__tabs {
+    display: flex;
+    border-bottom: 1px solid #a7aaad;
+
+    .components-button {
+      font-weight: bold;
+      display: inline-block;
+      flex-grow: 1;
+      text-align: center;
+      font-size: 16px;
+      padding: 16px;
+      height: auto;
+      border-radius: 0;
+      border-left: 1px solid #a7aaad;
```

Comment:
I would recommend relying on core colors for things like this:
https://github.com/Automattic/jetpack/blob/5a4ec0da4f32874708b4f7014690aa458ceca883/extensions/README.md#L279-L285